frontend/src/pages/Admin/Subcontractors/index.tsx

```diff
@@ -3,8 +3,16 @@ import { useFormControls } from "hooks";
 import { IFormControls } from "types";
 import { tableConfig } from "./tableConfig";
 import { FormConfig } from "./FormConfig";
+import useTitle from "hooks/useTitle";
+import { useEffect } from "react";
 
 export const Subcontractors = () => {
+  const { updateTitle } = useTitle();
+
+  useEffect(() => {
+    updateTitle("Subcontractors");
+  }, [updateTitle]);
+
   const formControls: IFormControls = useFormControls();
 
   return (
```

Comment:
this looks familiar. maybe if we get time after MVP, it could be dry-ed out, along with the ministries component above